### PR TITLE
Add DemandBase tracking code and update GA code

### PR DESCRIPTION
**Pull request by Padgett**
Reference CIA-189 for context

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by TrevorBramble** [comment]
I suspect Google is a few iterations away from implementing self-aware artificial intelligence in their analytics snippet...

@delivery approve

File: _templates/layout.html [hunk 1]
```diff
@@ -45,33 +45,17 @@
...
+  	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-  var trackOutboundLink = function(url) {
-    ga('send', 'event', 'outbound', 'click', url, {'hitCallback':
-      function () {
-        document.location = url;
-      }
-    });
-  };
```
Comment (cwebberOps) [inline _templates/layout.html:70]: So lines 64-70 need to actually stay.

**Comment by chef-delivery** [comment]
Change: 3fa506fc-7ab4-4145-a6e4-7419b3c89752 approved by: @TrevorBramble

**Pull request merged**